0200-mapping.Rmd

```diff
@@ -0,0 +1,158 @@
+
+
+# Mapping
```

Comment:
Data Collection and Mapping

really the mapping part is secondary to knowing where to go and how to input the data. Field crews don't need to know how to make maps vs navigate and collect data. we can include some more techinical qgis but should start with low hanging fruit